quality/metrics/chrF.md

```diff
@@ -0,0 +1,27 @@
+---
+grand_parent: Building and research
+parent: Metrics
+nav_order: 5
+title: chrF
+description: CHaRacter-level F-score
+---
+
+**chrF** (CHaRacter-level F-score) was introduced in 2015 by Maja Popovic.
```

Comment:
Maja Popović

Reply:
The first sentence should probably introduce what's salient about the metric.

In this case, I think that it's character-based.

> **chrF** is a metric for machine translation evaluation that calculates the similarity between a machine translation output and a reference translation using character n-grams, not word n-grams.

Reply:
>
> Metrics based on word n-grams are especially problematic for high-morphology languages.
>